src/Device/Driver/AR62xx.cpp

```diff
@@ -0,0 +1,573 @@
+/*
+Copyright_License {
+
+  XCSoar Glide Computer - http://www.xcsoar.org/
+  Copyright (C) 2000-2016 The XCSoar Project
+  A detailed list of copyright holders can be found in the file "AUTHORS".
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+}
+*/
+
+#include "Device/Driver/AR62xx.hpp"
+#include "Device/Driver.hpp"
+#include "Device/Port/Port.hpp"
+#include "NMEA/Info.hpp"
+#include "RadioFrequency.hpp"
+#include "Thread/Cond.hxx"
+#include "Thread/Mutex.hxx"
+#include "Util/CharUtil.hxx"
+#include "Util/StaticFifoBuffer.hxx"
+#include "Util/Compiler.h"
+#include "LogFile.hpp"
+
+#include <cstdint>
+
+#include <stdio.h>
+
+/** max command length to send to AR62xx*/
+const int MAX_CMD_LEN = 128;
+
+/**
+ * active / passive station
+ * the ar62xx can hold 2 stations in parallel
+ * 1 = active station of AR62xx
+ * 0 = passive station of AR62xx
+ */
+const int ACTIVE_STATION = 1;
+const int PASSIVE_STATION = 0;
```

Comment:
Why are these global variables exported?
I guess this should be an `enum`.

Reply:
moved to private - should not be an enum